homeassistant/components/tractive/device_tracker.py

```diff
@@ -97,7 +96,6 @@ def battery_level(self):
     async def async_added_to_hass(self):
         """Handle entity which will be added."""
 
-        @callback
```

Comment:
We need to keep this decorator to make sure the function runs in the event loop. My previous comment was that we can move them to be instance methods instead of separate functions. That way we don't need to define them on each entity addition.